### PR TITLE
Add support for PhpStorm

### PR DESCRIPTION
**Pull request by atomicptr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Supports:
 - CLion (Toolbox, Snap)
 - GoLand (Toolbox)
 - PyCharm (Toolbox)
+- PhpStorm (Toolbox)
```
Comment (ggothe) [inline README.md:17]: if this was my repo i would ask you to sort lists alphabetically :grin:

Comment (lunaryorn) [reply README.md:17]: It's not your repo, though.

**Comment by lunaryorn** [comment]
@atomicptr Thanks :pray:

**Comment by lunaryorn** [comment]
@atomicptr Made a release, but mandatory review on Gnome Extensions takes a while, and sometimes a long time.  Meanwhile you can get the ZIP from Github releases here.

**Pull request merged**